package.json

```diff
@@ -34,7 +34,7 @@
   "homepage": "https://github.com/snyk/snyk-cocoapods-plugin#readme",
   "dependencies": {
     "@snyk/cli-interface": "^1.5.0",
-    "@snyk/cocoapods-lockfile-parser": "^2.0.0",
+    "@snyk/cocoapods-lockfile-parser": "^2.0.2",
```

Comment:
I would make all 3 of these exact (no ranges), so that we can control our releases

Reply:
by all 3 I mean:
```
@snyk/cli-interface
@snyk/cocoapods-lockfile-parser
@snyk/dep-graph (which should be 1.13.0)
```

Reply:
I guess, given how the snyk CLI is distributed, that does make sense.

Reply:
Really dep-graph 1.13.0? Snyk still ships with 1.12.0?

Reply:
It just adds a new method